### PR TITLE
fix(#449): add css base styles with class to positionable elements

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: surf-1570
fixes: https://github.com/rackerlabs/helix-ui/issues/449

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

## Before
Apr-25-2019 14-11-54-BEFORE

## After
Apr-25-2019 13-51-35

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit 07f5b27f4ce982349891b752f9875a466fe9951e

https://app.netlify.com/sites/helix-ui/deploys/5cc0cb597d5a3f000ba26ce9

File: src/helix-ui/styles/components/tooltips.less [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @import (reference) 'mixins';
 
-hx-tooltip {
+hx-tooltip:extend(.hxComponent) {
   #Positionable.base();
 
   color: @gray-900;
```
Comment (CITguy) [inline src/helix-ui/styles/components/tooltips.less]: Please remove `color` property. It's already inherited from `.hxComponent` styles.

File: src/helix-ui/styles/components.less [hunk 2]
```diff
@@ -1,3 +1,21 @@
+.hxComponent {
+  color: @gray-900;
+  font-family: "Roboto", sans-serif;
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: normal;
+  line-height: 1.5;
+  text-align: initial;
```
Comment (CITguy) [inline src/helix-ui/styles/components.less]: `initial` -> `left`

File: docs/components/menus/test.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <section>
+    <header>
+      <h2 id="menu-in-table-header">Menu Inside a Table Header</h2>
+    </header>
+    <table class="hxTable">
+      <thead>
+        <tr>
+          <th>
+            Table Header
+            <hx-disclosure class="hxBtn hxCog" aria-controls="cogMenuId">
```
Comment (100stacks) [inline docs/components/menus/test.html]: please alphabetize (or use the same attribute order)

File: docs/components/popovers/test.html [hunk 4]
```diff
@@ -529,4 +527,33 @@ <h3>Deprecated Scrolling Content</h3>
...
+  <section>
+    <header>
+      <h2 id="popover-in-table-header">Popover Inside a Table Header</h2>
+    </header>
+    <table class="hxTable">
+      <thead>
+        <tr>
+          <th>
+            Table Header
+            <hx-disclosure aria-controls="testPopover" open class="hxBtn">
```
Comment (100stacks) [inline docs/components/popovers/test.html]: please alphabetize (or use the same attribute order)

File: docs/components/tooltips/test.html [hunk 5]
```diff
@@ -243,4 +243,25 @@ <h2>Multiple Targets in Near Proximity</h2>
...
+    </header>
+    <table class="hxTable">
+      <thead>
+        <tr>
+          <th>
+            Rank
+            <p>
+              <hx-icon id="icon1" type="help-circle"></hx-icon>
+            </p>
+            <hx-tooltip open for="icon1" position="bottom-center">
```
Comment (100stacks) [inline docs/components/tooltips/test.html]: please alphabetize (or use the same attribute order)

**Pull request merged**